source/extensions/filters/http/jwt_authn/extractor.cc

```diff
@@ -330,10 +329,6 @@ absl::string_view ExtractorImpl::extractJWT(absl::string_view value_str,
   }
 
   // There should be two dots (periods; 0x2e) inside the string, but we don't verify that here
-  if (Runtime::runtimeFeatureEnabled("envoy.reloadable_features.token_passed_entirely")) {
-    return value_str.substr(starting);
```

Comment:
Sorry, I don't get something fundamental in this PR.
In #28678 you introduced a feature, and added a runtime-guard around it, so that if one wants to temporarily disable it, they can.
This PR, IIUC, is removing the implementation that was added (reverting the previous PR). Why is that?

Reply:
Yes! you're right I missed it that runtime guard is by default enabled.
Now I fixed it and now it will always run my change.
Till now code is cutting the valid token and from now always pass it as is.
Thanks for your important comment!